docs/ml/upload-model.md

```diff
@@ -8,46 +8,81 @@ aliases:
   - /manage/data/upload-model/
   - /manage/ml/upload-model/
 description: "Upload a Machine Learning model to Viam to use it with the ML Model service."
-# SME: Aaron Casas
+# SME: Steven B.
 ---
 
-On the [**DATA** tab](https://app.viam.com/data/view) in the Viam app, navigate to the **Models** subtab.
+## Upload an existing model to the registry
```

Comment:
This page needs an introduction for the two parts to this? Users may land here and not have uploaded their model to Viam yet. The second part of this page should almost certainly come first because it's the more general case. This one is really more "if you later want to make a model public you can". And the title there shouldn't be uploading (it's already uploaded), it should be `Make and exisitng model public in the registry` or something like that